internal/version/version.go

```diff
@@ -19,9 +19,11 @@ import (
 	"github.com/cri-o/cri-o/utils"
 )
 
+// Version is the version of the build.
+const Version = "1.18.0-rc1"
```

Comment:
should this be `1.18.1`?

Reply:
Actually let's make it `1.18.0-rc2` instead.

Reply:
fixed